broker/aws/api.go

```diff
@@ -0,0 +1,22 @@
+package aws
+
+import (
+	"errors"
+
+	"github.com/Symantec/Dominator/lib/log"
```

Comment:
Why not use standard library's `log` package? What are the benefits using the lib from Dominator?